impl/src/main/java/io/kroxylicious/testing/kafka/testcontainers/TestcontainersKafkaCluster.java

```diff
@@ -250,16 +254,27 @@ private KafkaContainer buildKafkaContainer(KafkaClusterConfig.ConfigHolder holde
         copyHostKeyStoreToContainer(kafkaContainer, properties, SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG);
 
         kafkaContainer
+                // ozangunalp style
                 .withEnv("SERVER_PROPERTIES_FILE", "/cnf/server.properties")
+                // ozangunalp style
                 .withEnv("SERVER_CLUSTER_ID", holder.kraftClusterId())
+                // apache style
+                .withEnv("CLUSTER_ID", holder.kraftClusterId())
                 // disables automatic configuration of listeners/roles by kafka-native
                 .withEnv("SERVER_AUTO_CONFIGURE", "false")
+                // ozangunalp style
                 .withCopyToContainer(Transferable.of(propertiesToBytes(properties), 0644), "/cnf/server.properties")
+                // apache style
+                .withCopyToContainer(Transferable.of(propertiesToBytes(properties), 0644), "/etc/kafka/docker/server.properties")
                 .withStartupAttempts(CONTAINER_STARTUP_ATTEMPTS)
                 .withMinimumRunningDuration(MINIMUM_RUNNING_DURATION)
                 .withStartupTimeout(STARTUP_TIMEOUT);
 
         if (clusterConfig.isSaslScram() && !clusterConfig.getUsers().isEmpty()) {
+            if (clusterConfig.isKafkaPostVersion4()) {
+                throw new RuntimeException("scram not implemented for apache kafka");
+            }
+            // todo get scram users into apache native
```

Comment:
wondering, do we need to support the ozan-native-style configuration where we format the storage with scram users, so that nodes can uses scram auth for intercommunication? Should we instead use the admin client to create the scram creds after the cluster is running?

Reply:
> Should we instead use the admin client to create the scram creds after the cluster is running?

I think so, let's use the Kafka public APIs everywhere.